application/cms/models.py

```diff
@@ -242,7 +242,10 @@ def __lt__(self, other):
     #                                               (latest created, not latest published, so could be a new draft)
 
     review_token = db.Column(db.String())  # used for review page URLs
-    description = db.Column(db.Text)  # Short description aimed at search result listings and social media sharing
+    description = db.Column(db.Text)  # A short summary used by search engines and social sharing.
+    additional_description = db.Column(db.TEXT)  # TOPIC PAGES ONLY: short paragraph displayed on topic page itself
```

Comment:
We've removed these columns (additional_description, page_type, position) from the Python model now, so don't think we want to add them back in?

Reply:
@samuelhwilliams ah yeah, think that was just a bad automatic merge. Will fix.